src/main/kotlin/com/manimdsl/runtime/ASTExecutor.kt

```diff
@@ -404,7 +405,7 @@ class VirtualMachine(
                         }
                     linearRepresentation.addAll(
                         listOf(
-                            ArrayElemRestyle(arrayIdent, listOf(index1, index2), newObjectStyle),
+                            ArrayElemRestyle(arrayIdent, listOf(index1, index2), newObjectStyle, true),
```

Comment:
Instead of is being set at true, can we put this property in the stylesheet? So that the user can decide whether he wants arrays (or a specific array) to have pointers when accessing or not?